src/btree/bt_curprev.c

```diff
@@ -789,16 +812,18 @@ __wt_btcur_prev(WT_CURSOR_BTREE *cbt, bool truncating)
             }
             if (ret != WT_NOTFOUND)
                 break;
-            /*
-             * If we are doing a search near with prefix key configured or the cursor has bounds
-             * set, we need to check if we have exited the prev function due to a prefix key
-             * mismatch or the key is out of bounds. If so, we break instead of walking onto the
-             * next page. We're not directly returning here to allow the cursor to be reset first
-             * before we return WT_NOTFOUND.
-             */
-            if (key_out_of_bounds)
-                break;
         }
+
+        /*
+         * If we are doing a search near with prefix key configured or the cursor has bounds set, we
```

Comment:
bt_curprev has no notion of prefix search so please remove it from this comment.